### DISCUSSION TITLE
A small visual improvement for the mobile sidebar hamburger UI/UX

### DISCUSSION THREAD

**Discussion by SoumyaK4** [cotes2020/jekyll-theme-chirpy#2059] [Ideas]
### Description

While transitioning my [Go Game Guide](https://weiqi.soumyak4.in/) to the Chirpy Theme, one of my UI/UX Designer friend pointed this out for the mobile UI/UX. That by default this doesn't become a 'X' icon when the menu is open in mobile.
![image](https://github.com/user-attachments/assets/8ade6fbc-6721-4316-9258-af7b0d9c03bf) 
I added a few lines in  `assets/css/jekyll-theme-chirpy.scss`
As I literally have no experience in collaboration, and also this theme uses a lot of technologies, I don't want to make a mess and give you devs a headache by creating PRs
[Here's the code](https://github.com/BB-Dojo/weiqi/blob/6644283e309085e67f7b0810e9beacb431966969/assets/css/jekyll-theme-chirpy.scss#L16)
```
body[sidebar-display] {
    .fa-bars:before,
    .fa-navicon:before {
      content: '\58';
    }
```

With my limited knowledge I think it could be easily incorporated in the code base, hoping it would be there in the next release.
@cotes2020 @kungfux 
Have a good day :heart:

**Comment by SoumyaK4** [comment]
hey @kungfux I just stumbled around this idea
![image](https://github.com/user-attachments/assets/c75ffcf0-bd7a-40ac-949d-ff5d6610b919)
we can have something like this shown from the _tabs collection. It could be a feature specefic to PWA because people who want a PWA are definitely going for a more app-like look.

If no of tabs > 5 
it can be hidden within the Menu Hamburger here, and it should act like the current mobile menu hamburger.

**Optional**
There could be a place in `_config.yml` where people can address the main 4 mobile menu items.

**Comment by cotes2020** [comment]
Hi @SoumyaK4, thank you for sharing this idea, but unfortunately, I regret to inform you that I do not plan to adopt it.

For specific reasons, please see: https://github.com/cotes2020/jekyll-theme-chirpy/pull/2072#pullrequestreview-2471059702

**Reply by SoumyaK4** [reply]
@cotes2020 understood 
Btw any hope for the mobile nav-bar at the bottom, that I shared [here](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2059#discussioncomment-11387821)?

**Reply by cotes2020** [reply]
Frankly, there are no such plans. As phone screens get larger, many browsers have fixed the address bar at the bottom of the screen, so adding another Chirpy navigation bar would look like a sandwich.

It is important to note that PWA is intended to speed up page rendering by avoiding repeated requests for previously visited pages, not to turn it into an app. After all, web pages and apps are designed differently, and the web design is subject to many browser limitations, such as gesture operations and the bottom address bar.

**Comment by SoumyaK4** [comment]
Aha
Thank you for explaining 
I've started learning about pwa a few days ago, and honestly @cotes2020 you explained it better than those google videos 🤭
As I practice more I'll probably make a custom mobile nav bar in the near future for my site.
Thank you for responding, 
Much love ❤️
